lib/rules/selector-combinator-space-after/__tests__/index.js

```diff
@@ -415,8 +518,61 @@ testRule(rule, {
   reject: [
     {
       code: "a+  a {}",
+      fixed: "a+ a {}",
       description: "two spaces after + combinator",
       message: messages.expectedAfter("+")
     }
   ]
 });
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: ["always"],
+  fix: true,
+
+  accept: [
```

Comment:
Can you move the tests in this block into the first `testRule` block (the one without any `syntax: property`), please?

The syntax is valid draft spec so [belongs](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#best-practices) in the first `testRule`.